### PR TITLE
ASC-1654 error on empty objects in lists for config

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: zigzag/zigzag_config.py [hunk 1]
```diff
@@ -60,6 +60,8 @@ def get_config(self, config_name, test_log=None):
             ZigZagConfigError
         """
 
+        error_message = "The config '{}' was not found in the config file".format(config_name)
```
Comment (ryan-rs) [inline zigzag/zigzag_config.py]:
```suggestion
        error_message = "The config setting '{}' was not found in the config file".format(config_name)
```

File: tests/unit/test_zigzag_config_file.py [hunk 2]
```diff
@@ -101,6 +101,52 @@ def config_missing_project_id_key(tmpdir_factory):
...
+
+    with open(str(config_path), 'w') as f:
+        f.write(config)
+
+    return config_path
+
+
+@pytest.fixture(scope='session')
+def empty_module_hierarchy(tmpdir_factory):
+    """config with one value in a jinga template"""
```
Comment (ryan-rs) [inline tests/unit/test_zigzag_config_file.py]: Description doesn't match intention of fixture.

File: tests/unit/test_zigzag_config_file.py [hunk 3]
```diff
@@ -101,6 +101,52 @@ def config_missing_project_id_key(tmpdir_factory):
     return config_path
 
 
+@pytest.fixture(scope='session')
+def module_hierarchy_two_nodes(tmpdir_factory):
+    """config with one value in a jinga template"""
```
Comment (ryan-rs) [inline tests/unit/test_zigzag_config_file.py]: Description doesn't match intention of fixture.

Comment (zreichert) [reply tests/unit/test_zigzag_config_file.py]: Updated

File: tests/unit/test_zigzag_config_file.py (hunk 2 above)
Comment (zreichert) [reply tests/unit/test_zigzag_config_file.py]: Updated

**Pull request merged**